nixos/modules/system/boot/networkd.nix

```diff
@@ -877,6 +877,7 @@ let
           "IAID"
           "RequestBroadcast"
           "RouteMetric"
+          "RapidCommit"
```

Comment:
[Docs](https://www.freedesktop.org/software/systemd/man/latest/systemd.network.html#RapidCommit=) (for anyone coming on this PR later, no need to update the PR for this.)